content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,27 @@
+public: yes
+location: open-source
+grid_template: true
+image:
+  - src: 'blog/2016/coachhub/feature.jpg'
+project:
+  - name: 'Sass Accoutrement'
+    tagline: 'Modular theme-configuration libraries'
+    url: '/accoutrement'
+    source: 'https://github.com/oddbird/?utf8=%E2%9C%93&query=+only%3Apublic+accoutrement'
+    status: 'public'
+    owner: yes
+contributors:
+  - author: 'miriam'
+    role: 'creator'
+summary: |
+  A set of related theme-configuration libraries
+  for building and documenting style patterns
+  directly in your Sass code,
+  as part of the development process.
+  Currently includes modules for managing colors,
+  webfonts and typography, modular scales,
+  and responsive layouts.
```

Comment:
Am I right that these summaries aren't used? At least I can't figure out how to navigate to them.